_posts/2017-06-21-Devoxx-pl.md

```diff
@@ -0,0 +1,104 @@
+---
+layout: post
+authors: [bart_blommaerts]
+title: "Devoxx Poland 2017"
+image: /img/2017-devoxx-pl/devoxx-poland.jpg
+tags: []
+category: Conference
+comments: true
+---
+# Devoxx Poland 2017
+(This post is still under progress and will be updated regularly)
+
+Krakow in the ICE Krakow Congress Centre.
+
+![ICE Krakow Congress Centre](/img/2017-devoxx-pl/venue-1.jpg){: .image width="65%" }
+![ICE Krakow Congress Centre foyer](/img/2017-devoxx-pl/venue-2.jpg){: .image width="34%" }
+
+We started off day 1 with the keynote in the beautiful main room.
+
+![Main room 1](/img/2017-devoxx-pl/room-1.jpg){: .image .fit }
+
+## Keynote: Speed without Discipline: a Recipe for Disaster 
+by [Venkat Subramaniam](https://twitter.com/@venkat_s)
+
+![Venkat Subramaniam - Speed without Discipline: a Recipe for Disaster](/img/2017-devoxx-pl/Venkat-Subramaniam-Speed-without-Discipline-a-Recipe-for-Disaster.jpg){: .image .fit }
+
+Paradigm shift right now
+- 90's -> object related programming (before procedures)
```

Comment:
In the nineties, everybody was doing imperative programming, using objects to implement functionality.

Maybe use full sentences?